### ISSUE TITLE
full CBL build on ubuntu 14.04 fails on estscan

### ISSUE THREAD

**Issue by andersje** [chapmanb/cloudbiolinux#163]
estscan seems to be problematic, along with a missing perlapi package:

INFO: Package install progress: 0/294
[localhost] local: sudo http_proxy=$http_proxy https_proxy=$https_proxy  bash -c "apt-get -y --force-yes install ec2-ami-tools ec2-api-tools euca2ools python-vm-builder-ec2 s3cmd bio-linux-clustal bio-linux-dotter bio-linux-jalview bio-linux-mcl bio-linux-mummer bio-linux-pfaat bio-linux-prank bio-linux-seaview bio-linux-squint bio-linux-t-coffee exonerate mafft muscle probcons wise bio-linux-artemis bio-linux-big-blast bio-linux-cd-hit bio-linux-estscan bio-linux-jprofilegrid bio-linux-rbs-finder bio-linux-transterm-hp bio-linux-trnascan mcl predictprotein"
sudo: unable to resolve host ip-10-223-0-31
Reading package lists... Done
Building dependency tree  
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 bio-linux-estscan : Depends: estscan2 but it is not going to be installed
E: Unable to correct problems, you have held broken packages.

Fatal error: local() encountered an error (return code 100) while executing 'sudo http_proxy=$http_proxy https_proxy=$https_proxy  bash -c "apt-get -y --force-yes install ec2-ami-tools ec2-api-tools euca2ools python-vm-builder-ec2 s3cmd bio-linux-clustal bio-linux-dotter bio-linux-jalview bio-linux-mcl bio-linux-mummer bio-linux-pfaat bio-linux-prank bio-linux-seaview bio-linux-squint bio-linux-t-coffee exonerate mafft muscle probcons wise bio-linux-artemis bio-linux-big-blast bio-linux-cd-hit bio-linux-estscan bio-linux-jprofilegrid bio-linux-rbs-finder bio-linux-transterm-hp bio-linux-trnascan mcl predictprotein"'

Aborting.

I've tried all the other packages in that list, and they all install just fine, so it appears to be with the bio-linux-estscan.

searching for, and trying to manually install estscan2 gets this chain of complaints:

ubuntu@ip-10-223-0-31:~/cloudbiolinux$ apt-cache search estscan2
estscan2 - detects coding regions of DNA sequences
libbtlib-perl - library for basic sequence manipulation
bio-linux-estscan - detects coding regions of DNA sequences.
ubuntu@ip-10-223-0-31:~/cloudbiolinux$ sudo apt-get install estscan2
sudo: unable to resolve host ip-10-223-0-31
Reading package lists... Done
Building dependency tree  
Reading state information... Done
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 estscan2 : Depends: perlapi-5.14.2 but it is not installable
            Depends: libbtlib-perl but it is not going to be installed
E: Unable to correct problems, you have held broken packages.
ubuntu@ip-10-223-0-31:~/cloudbiolinux$ sudo apt-cache search perlapi
sudo: unable to resolve host ip-10-223-0-31
perl-base - minimal Perl system
libperl-apireference-perl - Perl module to programmatically query the perlapi
ubuntu@ip-10-223-0-31:~/cloudbiolinux$ dpkg --list | grep perl-base
ii  perl-base                                5.18.2-2ubuntu1                       amd64        minimal Perl system
ubuntu@ip-10-223-0-31:~/cloudbiolinux$ dpkg --list | grep libperl-apireference
ii  libperl-apireference-perl                0.16-1                                all          Perl module to programmatically query the perlapi

So, basically, it LOOKS like estscan2 should be able to install, but it's got a bad dependency in there, or maybe perl-base doesn't really satisfy the package.
